README.md

```diff
@@ -182,3 +182,20 @@ result in build failure.
 All files in the repository must contain a license header in the format mentioned in [License Header](license.header).
 
 The static code analysis framework will also validate if the license exists in the specified format.
+
+### Xpanse Security Issues
+
+The xpanse project involves the use of some sensitive information, such as AK/SK sensitive fields in
+credential management, Sensitive variable information during service deployment, etc.
+
+If there is no AES encrypted file aes_sec in the root path of the project, the above sensitive
+information will not be encrypted and decrypted using AES encryption
+
+If you need to encrypt your sensitive information, please generate an AES file (file name: aes_sec)
+under the project root path. The content of the AES key file is the encryption key 256 value
+generated by visiting https://acte.ltd/utils/randomkeygen
```

Comment:
Add a new sub section 
### Generate AES Private Key
AES private key can be generated by any available tools which generates  random 256 bit key.  

Example - Use https://acte.ltd/utils/randomkeygen to generate the key and copy the value in `Encryption key 256Encryption key 256` field to the `aes_sec` file.